.gitignore

```diff
@@ -22,6 +22,8 @@
 /pkg/
 /tmp/
 /tools/
+/.idea
+/.vscode
```

Comment:
Hmm, this rather belongs into your global config, doesn't it? Adding every editor's temporary files to every project is too much clutter IMHO.

Reply:
OK, then I will remove them

Reply:
I agree that this should be part of each developer's global config. There then is much more IDEA and OS folders to be gitignored specific for each developer, such as .DS_Store, and I don't feel like getting into this rabbit hole